public/lesswrong-editor/src/ckeditor.js

```diff
@@ -125,21 +125,18 @@ const postEditorConfig = {
 		'|',
 		'link',
 		'|',
-		'comment',
-		'|',
-		'mathpreview'
+    'mathpreview',
+    'comment',
 	],
 	image: {
 		toolbar: [
-      'imageStyle:alignLeft',
-      'imageStyle:alignCenter',
-      'imageStyle:alignRight',
-      'imageStyle:full',
-			'|',
-			'imageTextAlternative',
-			'|',
+      // 'imageStyle:alignLeft',
```

Comment:
Why are these commented out now?

Reply:
Oli didn't want image positioning to avoid users creating weirdly-formatted-mobile content

Reply:
(deleted the comments now)